exporter/dynatraceexporter/README.md

```diff
@@ -24,6 +24,10 @@ The Dynatrace exporter is enabled by adding a `dynatrace` entry to the `exporter
 All configurations are optional, but if an `endpoint` other than the OneAgent metric ingestion endpoint is specified then an `api_token` is required.
 To see all available options, see [Advanced Configuration](#advanced-configuration) below.
 
+> When using this exporter, it is strongly RECOMMENDED to configure the OpenTelemetry SDKs to export metrics 
+> with DELTA temporality. If you are exporting Sum or Histogram metrics with CUMULATIVE temporality, read
+> about possible limitations of this exporter [here](#considerations-when-exporting-cumulative-data-points).
```

Comment:
```suggestion
> about possible limitations of this exporter [below](#considerations-when-exporting-cumulative-data-points).
```

Reply:
Thanks, updated in [f67a949](https://github.com/open-telemetry/opentelemetry-collector-contrib/pull/8848/commits/f67a94972a47328f059b316c1059f907bacba1fc)! :slightly_smiling_face: